tests/models/utils.py

```diff
@@ -47,3 +48,28 @@ def assert_second_fit_refits_model(
     reco_i2i_1 = model_1.recommend_to_items(dataset.item_id_map.external_ids, dataset, k, False)
     reco_i2i_2 = model_2.recommend_to_items(dataset.item_id_map.external_ids, dataset, k, False)
     pd.testing.assert_frame_equal(reco_i2i_1, reco_i2i_2, atol=0.001)
+
+
+def assert_default_config_and_default_model_params_are_the_same(
+    model: ModelBase, default_config: tp.Dict[str, tp.Any]
+) -> None:
+    model_from_config = model.from_config(default_config)
+    assert model_from_config.get_config() == model.get_config()
+
+
+def assert_get_config_and_from_config_compatibility(
+    model: ModelBase, dataset: Dataset, initial_config: tp.Dict[str, tp.Any], simple_types: tp.Optional[bool] = None
```

Comment:
1. No sense to make it `simple_types: tp.Optional[bool] = None`, it's better to make it `simple_types: bool = False` as it is by default in `get_config`, then you don't need `if simple_types is not None ...`
2. It's better to check both cases for all models (including your new models), so you never need `None`

Reply:
Fair point, added simple_types support into each model test.